### PR TITLE
apply storybook-dark-mode

### PR DESCRIPTION
**Pull request by ujw0712**
## Description

Apply dark mode to a storybook doc using the [storybook-dark-mode](https://storybook.js.org/addons/storybook-dark-mode)

https://github.com/dooboolab/dooboo-ui/issues/235#issue-1571744801

## Tests

storybook-dark-mode-test

## Checklist

- [x] yarn add `storybook-dark-mode`
- [x] Accordion
- [x] Button
- [x] ButtonGroup
- [x] Checkbox
- [x] FAB
- [x] Icon
- [x] IconButton
- [x] LoadingIndicator
- [x] ProgressBar
- [x] ProgressCircle
- [x] RadioGroup
- [x] SwitchToggle

### CODE REVIEW

File: docs/.storybook/darkTheme.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ export default create({
 
   // UI
   appBg: 'black',
-  appContentBg: '#111',
+  // appContentBg: '#111',
```
Comment (hyochan) [inline docs/.storybook/darkTheme.js:14]: Looks like you can remove this :)

File: docs/.storybook/main.js [hunk 2]
```diff
@@ -1,9 +1,6 @@
 module.exports = {
-  "stories": [
-    "../src/**/*.stories.mdx",
-    "../src/**/*.stories.@(js|jsx|ts|tsx)"
-  ],
-  "addons": [
+  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
```
Comment (hyochan) [inline docs/.storybook/main.js:3]: Why changing indentation here?

File: docs/.storybook/preview.js [hunk 3]
```diff
@@ -42,4 +64,10 @@ addParameters({
     hierarchyRootSeparator: /\|/,
     panelPosition: 'bottom',
   },
-});
+  darkMode: {
+    dark: {...themes.dark},
+    light: {...themes.light},
```
Comment (hyochan) [inline docs/.storybook/preview.js:69]: Above looks reduntant 
```
dark: themes.dark,
light: themes.light,
```

File: docs/src/components/Button/Outlined.tsx [hunk 4]
```diff
@@ -1,77 +1,70 @@
...
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
+        <Button
+          color="light"
+          text="Light"
+          type="outlined"
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
```
Comment (hyochan) [inline docs/src/components/Button/Outlined.tsx:66]: Looks like we can `map` here

File: docs/src/components/Button/Sizes.tsx [hunk 5]
```diff
@@ -1,53 +1,43 @@
...
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
+        <Button
+          color="primary"
+          size="small"
+          text="Primary"
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
```
Comment (hyochan) [inline docs/src/components/Button/Sizes.tsx:39]: Same here 😀

File: docs/src/components/Button/Solid.tsx [hunk 6]
```diff
@@ -1,72 +1,62 @@
...
+          text="Info"
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
+        <Button
+          color="light"
+          text="Light"
+          style={{margin: 8}}
+          onPress={() => {}}
+        />
```
Comment (hyochan) [inline docs/src/components/Button/Solid.tsx:58]: Same here

File: docs/src/components/Checkbox/EndElement.tsx [hunk 7]
```diff
@@ -1,62 +1,68 @@
...
+          <Checkbox
+            checked={checked}
+            onPress={() => setChecked(!checked)}
+            endElement={<StyledText>Hello this is a checkbox</StyledText>}
+          />
+          <Checkbox
+            checked={checked}
+            onPress={() => setChecked(!checked)}
+            endElement={<StyledText>Hello this is a checkbox</StyledText>}
+          />
```
Comment (hyochan) [inline docs/src/components/Checkbox/EndElement.tsx:63]: Here too

File: docs/src/components/RadioGroup/Colored.tsx [hunk 8]
```diff
@@ -1,76 +1,69 @@
...
+          selectedValue={curValue}
+          selectValue={setCurValue}
+        />
+        <RadioGroup
+          title="Danger"
+          type="danger"
+          data={data}
+          selectedValue={curValue}
+          selectValue={setCurValue}
+        />
```
Comment (hyochan) [inline docs/src/components/RadioGroup/Colored.tsx:65]: Same here

**Review by hyochan — APPROVED** [review]
Just a small style issue. Overall your work looks great 👍

## UI issue
Just note the UI issues with screenshots.
Screenshot 2023-02-25 at 1 31 38 AM

Screenshot 2023-02-25 at 1 31 23 AM

I'll handle the rest! Thanks a lot 💯

File: docs/.storybook/main.js (hunk 2 above)
Comment (ujw0712) [reply docs/.storybook/main.js:3]: Changed by 'prettier' module

**Pull request merged**